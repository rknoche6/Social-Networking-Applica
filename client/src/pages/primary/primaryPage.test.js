import React from "react";
import { shallow } from "enzyme";
import PrimaryPage from "./primaryPage";

describe("PrimaryPage", () => {
  let wrapper;

  beforeEach(() => (wrapper = shallow(<PrimaryPage />)));

  it("should render correctly", () => expect(wrapper).toMatchSnapshot());

  it("should render a <div />", () => {
    expect(wrapper.find("div").length).toEqual(1);
  });

  it("should render a <h1 />", () => {
    expect(wrapper.find("h1").length).toEqual(1);
  });
});