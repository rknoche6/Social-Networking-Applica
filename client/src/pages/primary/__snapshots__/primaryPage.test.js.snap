// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PrimaryPage should render correctly 1`] = `ShallowWrapper {}`;